flink-ai-flow/ai_flow_plugins/scheduler_plugins/airflow/dag_generator.py

```diff
@@ -86,9 +88,26 @@ def generate_op_code(self, job):
 job_{0} = json_utils.loads(job_json_{0})
 op_{0} = AIFlowOperator(task_id='{2}', job=job_{0}, workflow=workflow, dag=dag)
 """
-        return 'op_{}'.format(self.op_count), OP_DEFINE.format(self.op_count,
-                                                               json_utils.dumps(job),
-                                                               job.job_name)
+        OP_DEFINE_WITH_ARGS = """
+job_json_{0} = '{1}'
+job_{0} = json_utils.loads(job_json_{0})
+op_{0} = AIFlowOperator(task_id='{2}', job=job_{0}, workflow=workflow, dag=dag"""
```

Comment:
Seems that we don't need an OP_DEFINE_WITH_ARGS. OP_DEFINE  can be reused as long as handles the ')'.